gcc/testsuite/gcc.target/riscv/rvv/rvv-vget-vset.c

```diff
@@ -0,0 +1,27 @@
+/* { dg-do compile } */
+
+#include <riscv_vector.h>
+
+vint32m1_t vget(vint32m2_t data)
+{
+  return vget_v_i32m2_i32m1(data, 2);
+  /* { dg-error {index 2 is out of range, should be less than 2.} } */
+}
+
+vint32m1_t vget_tuple(vint32m1x4_t data)
+{
+  return vget_i32m1x4_i32m1(data, 4);
+  /* { dg-error {index 4 is out of range, should be less than 4.} } */
+}
+
+vint32m8_t vset(vint32m8_t data, vint32m1_t v)
+{
+  return vset_v_i32m1_i32m8(data, 8, v);
+  /* { dg-error {index 8 is out of range, should be less than 8.} } */
+}
+
+vint32m1x8_t vset_tuple(vint32m1x8_t data, vint32m1_t v)
+{
+  return vset_i32m1x8(data, 8, v);
+  /* { dg-error {index 8 is out of range, should be less than 8.} } */
```

Comment:
Still got failed even change this line to `/* { dg-error "index 8 is out of range, should be less than 8." "" { target *-*-* } 0 } */` 
I guess it because it's same as last `dg-error` check, but seems like change the testcase to 9 instead of 8 can resolve the issue.